### DISCUSSION TITLE
Is there any matrix of implemented/not implemented features by backend?

### DISCUSSION THREAD

**Discussion by cesarchefinho** [unoplatform/uno#9024] [Q&A]
Something like this 

Feature.                            Gtk.  Frame buffer     wpf   uwp.  Winui3   wasm 

Textbox.text.                    Yes.        Yes.              Yes.  Yes     yes.         Yes
Textbox.blabla.                 no.          Yes.             Yes.   No.    Yes.         No
WCT.Dropsahdow             No.         Yes.             Yes.    No.   Yes.         No
STANDALONE EXE.          Yes          No.               No.     No.   No.         No
Unotk.Elevatedview.         Yes.          Yes.             Yes.    Yes. Yes.        Yes

And

What is the % of implemented features in all backends

Somethings like comparing wpf and gtk backends, features not implemented etc...

**Comment by jeromelaban** [comment]
Thanks. For the time being, the only page similar to this is [the implemented views.](https://platform.uno/docs/articles/implemented-views.html).

**Reply by cesarchefinho** [reply]
> Thanks. For the time being, the only page similar to this is [the implemented views.](https://platform.uno/docs/articles/implemented-views.html).

Thanks. The problem with implemented views article is that it not cover all backends and don't have any totalization of implemented features.

I think if this information are available to developers it will facilitate the decision of the backend used to develop new aplications.

Its only a suggestion.

Thank you for response.
